### PR TITLE
Aerogear-9364 - Follow up on comments for showcase_app PR #682

### PR DESCRIPTION
**Pull request by jstaffor**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-9364

## What
https://issues.jboss.org/browse/AEROGEAR-9364

## Why
https://issues.jboss.org/browse/AEROGEAR-9364

## How
https://issues.jboss.org/browse/AEROGEAR-9364

## Verification Steps
https://issues.jboss.org/browse/AEROGEAR-9364

### CODE REVIEW

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 1]
```diff
@@ -109,9 +109,9 @@ xref:setting-up-mobile-services-to-demonstrate-showcase-apps[Setting up Mobile S
 		}
 	}]
 }
-
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: Validated here: https://github.com/jstaffor/mobile-docs/pull/3#pullrequestreview-247507817

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 2]
```diff
@@ -39,11 +39,11 @@ For more information about features, see xref:identity-management.adoc[{keycloak
 
 . To demonstrate {unifiedpush-service}, xref:push-notifications.adoc#provisioning-push-notifications-service[Provision the {unifiedpush-service} service]. For more information about features, see xref:push-notifications.adoc[{unifiedpush-service} Service]
 
-// tag::excludeDownstream[]
+// tag::excludeUpstream[]
 
-NOTE: You can only demonstrate {sync-service} on Cordova.
+NOTE: You can only demonstrate {sync-service} on Cordova or Ionic.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: Hi @finp, my understanding is that the data sync can be built upon a Cordova or Ionic platform, and as such, I've included *Ionic* here - make sense to you?

File: modules/ROOT/pages/showcase-apps.adoc [hunk 3]
```diff
@@ -13,7 +13,7 @@ include::_partials/showcase-apps/index.adoc[leveloffset=1]
 
 
 // start showcase-apps/service-setup.adoc[leveloffset=1]
-include::_partials/showcase-apps/service-setup.adoc[leveloffset=2,tags=!excludeDownstream]
```
Comment (jstaffor) [inline modules/ROOT/pages/showcase-apps.adoc]: @finp, I've tested the tagging system usptream, and it now works as expected :+1:

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 4]
```diff
@@ -109,9 +109,9 @@ xref:setting-up-mobile-services-to-demonstrate-showcase-apps[Setting up Mobile S
...
 	}]
 }
-
-module.exports = config;
 ----
++
+. Copy the content to your clipboard.
 
 [[configuring-the-showcase-app-to-point-at-the-provisioned-data-sync-service]]
 === Configuring the Showcase App to point at the provisioned Data Sync service
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: @finp  - any proposals (https://github.com/aerogear/mobile-docs/pull/682#discussion_r291576080)? I can catch you at the stand-up tomorrow too if you want to talk it out...just let me know :+1:

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc (hunk 2 above)
Comment (finp) [reply modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: There are a few things here:
- a terminology question: Ionic is cordova, or at least our support for Ionic requires cordova, which is why we need to say 'Plain Cordova'
- I don't think this sentence and tagging make sense anymore... might be best to remove, create a jira to follow up?

Comment (jstaffor) [reply modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: That makes sense :+1:

Comment (jstaffor) [reply modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: https://issues.jboss.org/browse/AEROGEAR-9371

**Pull request merged**